packages/use-contractkit/src/screens/wallet-connect-custom.tsx

```diff
@@ -1,72 +1,107 @@
+import QrCode from 'qrcode.react';
 import React from 'react';
+import { isMobile } from 'react-device-detect';
 import Loader from 'react-loader-spinner';
 
 import { CopyText } from '../components';
 import { useWalletConnectConnector } from '../connectors/useWalletConnectConnector';
-import { WalletIds } from '../constants';
-import { Connector } from '../types';
+import { Connector, WalletEntry } from '../types';
 
 interface Props {
   onSubmit: (connector: Connector) => void;
+  wallet: WalletEntry;
 }
 
-export const CeloWallet: React.FC<Props> = ({ onSubmit }: Props) => {
+// TODO: get in touch with CeloWallet and see if we can harmonize deeplinks
+// CeloWallet: celowallet://wc?...
+// Valora/CeloDance: celo://wallet/wc?...
```

Comment:
The difference was intentional because users can have Celo Wallet and Celo Terminal both installed, so they need a unique scheme